src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -38,31 +37,6 @@ class ChaptersButton extends TextTrackButton {
   }
 
   /**
-   * Create a menu item for each text track
-   *
-   * @return {Array} Array of menu items
-   * @method createItems
-   */
-  createItems() {
```

Comment:
Dead code never called. Item creation was all placed inside `createMenu` for some reason.